README.md

```diff
@@ -25,16 +25,33 @@ bundle exec jekyll serve
 
 We are using the theme [Just the docs](https://github.com/pmarsceill/just-the-docs).
 
-Yo have to create a markdown file and follow the next structure:
+You have to create markdown files inside the folder struction, following the structure:
 
 ```
 ---
 layout: default
 title: A title
 parent: Name of the parent
 grand_parent: Name of the grand parent if needed
-has_child: true
+has_child: (bool)  
+has_toc: (bool) 
 ---
 
 Excepteur consequat reprehenderit velit mollit occaecat velit esse laborum eiusmod irure eu. Ex fugiat officia elit qui veniam nostrud nisi est. Est reprehenderit nisi sunt non nulla irure aute sunt do.
-```
\ No newline at end of file
+```
+
+Where: 
+
+- `has_child` links to child content in the folder structure
+- `has_toc` shows table of child content at the bottom of the page 
+
+## Making a PR
+
+Some tips & best practices (mainly for non-devs) when working on this repo:
```

Comment:
I suggest adding a label based on the FA. Also if you have conflicts on your brach prior to merging with master, please contact a developer.